contrib/sarplus/python/tests/test_pyspark_sar.py

```diff
@@ -0,0 +1,390 @@
+import calendar
+import datetime
+import math
+import numpy as np
+import pandas as pd
+import pytest
+import os
+from sklearn.model_selection import train_test_split
+
+from pyspark.sql import SparkSession
+
+from pysarplus import SARPlus, SARModel
+
+def assert_compare(expected_id, expected_score, actual_prediction):
+    assert expected_id == actual_prediction.id
+    assert math.isclose(expected_score, actual_prediction.score, rel_tol=1e-3, abs_tol=1e-3)
+
+@pytest.fixture(scope="module")
+def spark(app_name="Sample", url="local[*]", memory="1G"):
```

Comment:
Maybe reuse the initializer fixture function from the repo?

Reply:
not sure it's worth the dependency?

Reply:
It's a part of reco_utils. So here we don't want to have much reliance on the original reco repo, right?

Reply:
I think it makes the build processes easier (e.g. we'd have to check if other directories are changed as well), the yaml file only triggers on contrib/sarplus